### PR TITLE
Update de.m3u

### PR DESCRIPTION
**Pull request by Sphinxroot**
Added Some channel corrected logo

### CODE REVIEW

File: channels/de.m3u [hunk 1]
```diff
@@ -194,20 +194,40 @@ https://muenchentv.iptv-playoutcenter.de/muenchentv/muenchentv.stream_1/playlist
...
 https://ndrevent-lh.akamaihd.net/i/ndrevent_2@429805/master.m3u8
-#EXTINF:-1 tvg-id="NDREvent3.de" tvg-name="NDR Event 3" tvg-country="DE" tvg-language="" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/ndr.png" group-title="",NDR Event 3 (720p)
+#EXTINF:-1 tvg-id="NDREvent3.de" tvg-name="NDR Event 3" tvg-country="DE" tvg-language="German" tvg-logo="https://i.imgur.com/BuirhUR.png" group-title="",NDR Event 3 (720p)
 https://ndrevent-lh.akamaihd.net/i/ndrevent_3@409068/master.m3u8
-#EXTINF:-1 tvg-id="NDRFernsehen.de" tvg-name="NDR Fernsehen" tvg-country="DE" tvg-language="German" tvg-logo="https://i.imgur.com/UMaAaCW.png" group-title="",NDR Fernsehen (720p)
-https://ndrfs-lh.akamaihd.net/i/ndrfs_nds@430233/master.m3u8
-#EXTINF:-1 tvg-id="NDRHamburg.de" tvg-name="NDR Hamburg" tvg-country="DE" tvg-language="" tvg-logo="" group-title="",NDR Hamburg (720p)
+#EXTINF:-1 tvg-id="NDRFernsehen.de" tvg-name="NDR Fernsehen" tvg-country="DE" tvg-language="German" tvg-logo="https://www.ndr.de/fernsehen/livestreamteaserbild110_v-contentxl.jpg" group-title="",NDR Fernsehen (720p)
+https://mcdn.ndr.de/ndr/hls/ndr_fs/ndr_fs/master_720.m3u8
```
Comment (LaneSh4d0w) [inline channels/de.m3u:204]: Adaptive link : https://mcdn.ndr.de/ndr/hls/ndr_fs/ndr_fs/playlist.m3u8 (redirects to another Akamai server)

File: channels/de.m3u [hunk 2]
```diff
@@ -194,20 +194,40 @@ https://muenchentv.iptv-playoutcenter.de/muenchentv/muenchentv.stream_1/playlist
...
+#EXTINF:-1 tvg-id="NDRFernsehen.de" tvg-name="NDR Fernsehen" tvg-country="DE" tvg-language="German" tvg-logo="https://www.ndr.de/fernsehen/livestreamteaserbild110_v-contentxl.jpg" group-title="",NDR Fernsehen (720p)
+https://ndrfs-lh.akamaihd.net/i/ndrfs_fs@430230/master.m3u8
+#EXTINF:-1 tvg-id="NDRHamburg.de" tvg-name="NDR Hamburg" tvg-country="DE" tvg-language="German" tvg-logo="https://www.ndr.de/fernsehen/livestreamteaserbild100_v-contentxl.jpg" group-title="",NDR Hamburg (720p)
+https://mcdn.ndr.de/ndr/hls/ndr_fs/ndr_hh/master_720.m3u8
```
Comment (LaneSh4d0w) [inline channels/de.m3u:208]: Adaptive link : https://mcdn.ndr.de/ndr/hls/ndr_fs/ndr_hh/playlist.m3u8 (redirects to another Akamai server)

File: channels/de.m3u [hunk 3]
```diff
@@ -194,20 +194,40 @@ https://muenchentv.iptv-playoutcenter.de/muenchentv/muenchentv.stream_1/playlist
...
+#EXTINF:-1 tvg-id="NDRHamburg.de" tvg-name="NDR Hamburg" tvg-country="DE" tvg-language="German" tvg-logo="https://www.ndr.de/fernsehen/livestreamteaserbild100_v-contentxl.jpg" group-title="",NDR Hamburg (720p)
 https://ndrfs-lh.akamaihd.net/i/ndrfs_hh@430231/master.m3u8
-#EXTINF:-1 tvg-id="NDRMecklenburgVorpommern.de" tvg-name="NDR Mecklenburg-Vorpommern" tvg-country="DE" tvg-language="" tvg-logo="" group-title="",NDR Mecklenburg-Vorpommern (720p)
+#EXTINF:-1 tvg-id="NDRInternational.de" tvg-name="NDR International" tvg-country="DE" tvg-language="German" tvg-logo="https://www.ndr.de/fernsehen/livestreamteaserbild108_v-contentxl.jpg" group-title="",NDR International (720p)
+https://mcdn.ndr.de/ndr/hls/ndr_fs/ndr_int/master_720.m3u8
```
Comment (LaneSh4d0w) [inline channels/de.m3u:212]: Adaptive link : https://mcdn.ndr.de/ndr/hls/ndr_fs/ndr_int/playlist.m3u8 (redirects to another Akamai server)

File: channels/de.m3u [hunk 4]
```diff
@@ -194,20 +194,40 @@ https://muenchentv.iptv-playoutcenter.de/muenchentv/muenchentv.stream_1/playlist
...
+#EXTINF:-1 tvg-id="NDRInternational.de" tvg-name="NDR International" tvg-country="DE" tvg-language="German" tvg-logo="https://www.ndr.de/fernsehen/livestreamteaserbild108_v-contentxl.jpg" group-title="",NDR International (720p)
+https://ndrfs-lh.akamaihd.net/i/ndrfs_int@430236/master.m3u8
+#EXTINF:-1 tvg-id="NDRMecklenburgVorpommern.de" tvg-name="NDR Mecklenburg-Vorpommern" tvg-country="DE" tvg-language="German" tvg-logo="https://www.ndr.de/fernsehen/livestreamteaserbild102_v-contentxl.jpg" group-title="",NDR Mecklenburg-Vorpommern (720p)
+https://mcdn.ndr.de/ndr/hls/ndr_fs/ndr_mv/master_720.m3u8
```
Comment (LaneSh4d0w) [inline channels/de.m3u:216]: Adaptive link : https://mcdn.ndr.de/ndr/hls/ndr_fs/ndr_mv/playlist.m3u8 (redirects to another Akamai server)

File: channels/de.m3u [hunk 5]
```diff
@@ -194,20 +194,40 @@ https://muenchentv.iptv-playoutcenter.de/muenchentv/muenchentv.stream_1/playlist
...
+#EXTINF:-1 tvg-id="NDRMecklenburgVorpommern.de" tvg-name="NDR Mecklenburg-Vorpommern" tvg-country="DE" tvg-language="German" tvg-logo="https://www.ndr.de/fernsehen/livestreamteaserbild102_v-contentxl.jpg" group-title="",NDR Mecklenburg-Vorpommern (720p)
 https://ndrfs-lh.akamaihd.net/i/ndrfs_mv@430232/master.m3u8
-#EXTINF:-1 tvg-id="NDRSchleswigHolstein.de" tvg-name="NDR Schleswig-Holstein" tvg-country="DE" tvg-language="" tvg-logo="" group-title="",NDR Schleswig-Holstein (720p)
+#EXTINF:-1 tvg-id="NDRNiedersachsen.de" tvg-name="NDR Niedersachsen" tvg-country="DE" tvg-language="German" tvg-logo="https://www.ndr.de/fernsehen/livestreamteaserbild104_v-contentxl.jpg" group-title="",NDR Niedersachsen (720p)
+https://mcdn.ndr.de/ndr/hls/ndr_fs/ndr_nds/master_720.m3u8
```
Comment (LaneSh4d0w) [inline channels/de.m3u:220]: Adaptive link : https://mcdn.ndr.de/ndr/hls/ndr_fs/ndr_nds/playlist.m3u8 (redirects to another Akamai server)

File: channels/de.m3u [hunk 6]
```diff
@@ -194,20 +194,40 @@ https://muenchentv.iptv-playoutcenter.de/muenchentv/muenchentv.stream_1/playlist
...
+#EXTINF:-1 tvg-id="NDRNiedersachsen.de" tvg-name="NDR Niedersachsen" tvg-country="DE" tvg-language="German" tvg-logo="https://www.ndr.de/fernsehen/livestreamteaserbild104_v-contentxl.jpg" group-title="",NDR Niedersachsen (720p)
+https://ndrfs-lh.akamaihd.net/i/ndrfs_nds@430233/master.m3u8
+#EXTINF:-1 tvg-id="NDRSchleswigHolstein.de" tvg-name="NDR Schleswig-Holstein" tvg-country="DE" tvg-language="German" tvg-logo="https://www.ndr.de/fernsehen/livestreamteaserbild106_v-contentxl.jpg" group-title="",NDR Schleswig-Holstein (720p)
+https://mcdn.ndr.de/ndr/hls/ndr_fs/ndr_sh/master_720.m3u8
```
Comment (LaneSh4d0w) [inline channels/de.m3u:224]: Adaptive link : https://mcdn.ndr.de/ndr/hls/ndr_fs/ndr_sh/playlist.m3u8 (redirects to another Akamai server)

File: channels/de.m3u [hunk 7]
```diff
@@ -194,20 +194,40 @@ https://muenchentv.iptv-playoutcenter.de/muenchentv/muenchentv.stream_1/playlist
...
+#EXTINF:-1 tvg-id="NDRSchleswigHolstein.de" tvg-name="NDR Schleswig-Holstein" tvg-country="DE" tvg-language="German" tvg-logo="https://www.ndr.de/fernsehen/livestreamteaserbild106_v-contentxl.jpg" group-title="",NDR Schleswig-Holstein (720p)
 https://ndrfs-lh.akamaihd.net/i/ndrfs_sh@430234/master.m3u8
+#EXTINF:-1 tvg-id="NDRWeltweiter.de" tvg-name="NDR Weltweiter" tvg-country="DE" tvg-language="German" tvg-logo="https://www.ndr.de/fernsehen/livestreamteaserbild108_v-contentxl.jpg" group-title="",NDR Weltweiter (720p)
+https://mcdn.ndr.de/ndr/hls/ndr_fs/ndr_ww/master_720.m3u8
```
Comment (LaneSh4d0w) [inline channels/de.m3u:228]: Adaptive link : https://mcdn.ndr.de/ndr/hls/ndr_fs/ndr_ww/playlist.m3u8 (redirects to another Akamai server)

**Comment by Sphinxroot** [comment]
https://www.ndr.de/fernsehen/livestream/index.html  
yeah i see it redirect to the international stream but when i sniffed them it was`nt
it was  
https://mcdn.ndr.de/ndr/hls/ndr_fs/  host 

maybe not 24hr stream  they get active for global  event  or local  only

no it send you to

https://ndrint.akamaized.net/hls/live/2020766/ndr_int/master.m3u8
https://ndrint.akamaized.net/hls/live/2020766-b/ndr_int/master.m3u8

**Comment by LaneSh4d0w** [comment]
Yes, that's what I've seen.

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**